src/Relude/Extra/Map.hs

```diff
@@ -121,19 +121,45 @@ instance StaticMap IntSet where
     {-# INLINE lookup #-}
     {-# INLINE member #-}
```

Comment:
Should we refactor this so that each `INLINE` would go under the function?

Reply:
Good idea!